internal/confmapprovider/discovery/bundle/bundle.d/receivers/smartagent-collectd-kafka-broker.discovery.yaml.tmpl

```diff
@@ -0,0 +1,43 @@
+{{ receiver "smartagent/collectd/kafka_broker" }}:
+  enabled: true
+  rule:
+    docker_observer: type == "container" and any([name, image, command], {# matches "(?i)kafka.*broker"}) and not (command matches "splunk.discovery")
+    host_observer: type == "hostport" and command matches "(?i)kafka.*broker"
+    k8s_observer: type == "port" and pod.name matches "(?i)kafka.*broker"
+  config:
+    default:
+      type: collectd/kafka
+      extraMetrics: ["*"]
+      isolatedCollectd: true
+  status:
+    metrics:
+      successful:
+        - strict: counter.kafka-bytes-in
+          first_only: true
+          log_record:
+            severity_text: info
+            body: smartagent/collectd/kafka receiver is working!
+    statements:
+      failed:
+        - regexp: "Creating MBean server connection failed"
+          first_only: true
+          log_record:
+            severity_text: info
+            append_pattern: true
+            body: The container is unable to connnect to the MBean server to retrieve kafka metrics.
+        - regexp: "java.rmi.UnknownHostException: Unknown host"
+          first_only: true
+          log_record:
+            severity_text: info
+            append_pattern: true
+            body: Kafka host was discovered but is unresolvable.
+      partial:
+        - regexp: "clusterName is a required field"
```

Comment:
As a required field I think this will fail regardless of availability so it's a partial status imo.

I think the value is only used as an extra dimension, so could be a good candidate for different observer default values using using the endpoint environments. I'm not seeing a great approach w/ what's available though, so using the standard `default` helper is what I'd suggest.